src/plugins.js

```diff
@@ -0,0 +1,131 @@
+export default {
+  // es2015
+  "transform-es2015-arrow-functions": {
+    edge: 13,
+    firefox: 45,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-block-scoped-functions": {
+    edge: 11,
+    firefox: 46,
+    chrome: 41,
+    opera: 28,
+  },
+  "transform-es2015-block-scoping": {
+    edge: 11,
+    firefox: 36,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-classes": {
+    edge: 13,
+    firefox: 45,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-computed-properties": {
+    edge: 12,
+    firefox: 34,
+    chrome: 44,
+    opera: 31
+  },
+  "check-es2015-constants": {
+    edge: 11,
+    firefox: 36,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-destructuring": {
+    edge: 14,
+    firefox: Number.NEGATIVE_INFINITY,
+    chrome: 52,
+    opera: 39,
+  },
+  "transform-es2015-for-of": {
+    edge: Number.NEGATIVE_INFINITY,
+    firefox: Number.NEGATIVE_INFINITY,
+    chrome: 51,
+    opera: 38,
+  },
+  "transform-es2015-function-name": {
+    edge: Number.NEGATIVE_INFINITY,
+    firefox: Number.NEGATIVE_INFINITY,
+    chrome: 52,
+    opera: 39,
+  },
+  "transform-es2015-literals": {
+    edge: 12,
+    firefox: 36,
+    chrome: 41,
+    opera: 28
+  },
+  "transform-es2015-object-super": {
+    edge: 13,
+    firefox: 45,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-parameters": {
+    edge: 14,
+    firefox: Number.NEGATIVE_INFINITY,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-shorthand-properties": {
+    edge: 12,
+    firefox: 33,
+    chrome: 43,
+    opera: 30
+  },
+  "transform-es2015-spread": {
+    edge: 13,
+    firefox: 36,
+    chrome: 46,
+    opera: 33,
+  },
+  "transform-es2015-sticky-regex": {
+    edge: 13,
+    firefox: 31,
+    chrome: 49,
+    opera: 36
+  },
+  "transform-es2015-template-literals": {
+    edge: 13,
+    firefox: 34,
+    chrome: 41,
+    opera: 28,
+  },
+  "transform-es2015-typeof-symbol": {
+    edge: 12,
+    firefox: 36,
+    chrome: 38,
+    opera: 25,
+  },
+  "transform-es2015-unicode-regex": {
+    edge: 12,
+    firefox: 46,
+    chrome: 51,
+    opera: 38,
+  },
+  "transform-regenerator": {
+    edge: 13,
+    firefox: 45,
+    chrome: 49,
```

Comment:
This might still be needed in Chrome due to https://bugs.chromium.org/p/v8/issues/detail?id=5322

It works fine with defaults, but breaks with harmony enabled, or in Canary defaults.